mscore/inspector/inspectorImage.h

```diff
@@ -27,11 +27,15 @@ class InspectorImage : public InspectorElementBase {
       Q_OBJECT
 
       Ui::InspectorImage b;
+      qreal _ratio; // used for widthChanged/heightChanged
```

Comment:
I suggest that this be renamed to `_aspectRatio` for clarity.

Reply:
OK